graylog2-web-interface/webpack.config.js

```diff
@@ -132,11 +130,6 @@ if (TARGET === 'build') {
   process.env.NODE_ENV = 'production';
   module.exports = merge(webpackConfig, {
     plugins: [
-      new webpack.DefinePlugin({
```

Comment:
It is explained (here)[https://webpack.js.org/guides/production-build/#node-environment-variable]. The other usages of the `DefinePlugin` are used for something different, namely the red bubble in the web interface indicating that we are on dev, so we can keep them.